recipe/meta.yaml

```
@@ -8,31 +8,44 @@ package:
 source:
   url: https://ayera.dl.sourceforge.net/project/lcms/lcms/{{ version }}/lcms2-{{ version }}.tar.gz
   sha256: 48c6fdf98396fa245ed86e622028caf49b96fa22f3e5734f853f806fbc8e7d20
+  patches:
+        - remove_library_directive_win.patch  # [win]
+        - build_shared_win.patch  # [win]
 
 build:
-  number: 1
-  skip: True  # [win]
+  skip: true  # [win and vc<14]
+  number: 2
   run_exports:
     - {{ pin_subpackage("lcms2") }}
 
 requirements:
   build:
     - {{ compiler('c') }}
-    - make
+    - make  # [not win]
+    - cmake  # [win]
+
   host:
     - jpeg
     - libtiff
```

```suggestion
    - libtiff 4.0
```
but I don't think it will work as intended. I think it is easiest to create a patched up version of libtiff 4.1